### PR TITLE
update kb Dockerfile

### LINKED ISSUE #320: kb v0.5.0 is in CrashLoopBackOff status
**What happened**:
kb is in CrashLoopBackOff status

**What you expected to happen**:
kb is in Running status

**Anything else we need to know?**:
kb log:
```
Traceback (most recent call last):
  File "/code/lib/sedna/service/run_kb.py", line 18, in <module>
    from sedna.service.server.knowledgeBase.model import init_db
  File "/code/lib/sedna/__init__.py", line 16, in <module>
    import sedna.algorithms
  File "/code/lib/sedna/algorithms/__init__.py", line 19, in <module>
    from . import optical_flow
  File "/code/lib/sedna/algorithms/optical_flow/__init__.py", line 19, in <module>
    import cv2
ModuleNotFoundError: No module named 'cv2'
```

### PR DESCRIPTION
**Pull request by hey-kong**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
kb v0.5.0 requires opencv to run

**Which issue(s) this PR fixes**:

Fixes #320

### CODE REVIEW

File: build/kb/Dockerfile [hunk 1]
```diff
@@ -25,6 +28,7 @@ RUN pip install uvicorn~=0.14.0
 RUN pip install python-multipart~=0.0.5
 RUN pip install SQLAlchemy~=1.4.7
 RUN pip install aiofiles~=0.7.0
+RUN pip install opencv-python
```
Comment (JoeyHwong-gk) [inline build/kb/Dockerfile:31]: It is recommended that specify the version.

File: build/kb/Dockerfile [hunk 2]
```diff
@@ -14,6 +14,9 @@
 
 FROM python:3.6-slim
 
+RUN apt update \
+  && apt install -y libgl1-mesa-glx libglib2.0-0
```
Comment (JoeyHwong-gk) [inline build/kb/Dockerfile:18]: At the end of the dockerfile, clean up the apt cache by `apt clean` and `rm /var/lib/apt/`

**Comment by JoeyHwong-gk** [comment]
As a public dependency, it is recommended to extract OpenCV2  into the `reqirements.txt`.

**Comment by jaypume** [comment]
This dependency should be fixed to be imported when it is needed(Such as the example of Multi-Edge Inference).

**Comment by hey-kong** [comment]
> 

OK, I close this PR.

**Pull request closed without merging**